lib/src/stream_extensions.dart

```diff
@@ -6,6 +6,17 @@ import 'dart:async';
 
 /// Utility extensions on [Stream].
 extension StreamExtensions<T> on Stream<T> {
+
+  /// Creates a stream whose elements are contiguous slices of [this].
+  ///
+  /// Each slice is [length] elements long, except for the last one which may be
+  /// shorter if [this] emits too few elements. Each slice begins after the
+  /// last one ends.
+  ///
+  /// For example, `Stream.fromIterable([1, 2, 3, 4, 5]).slices(2)` emits `([1,
```

Comment:
I don't think you can break code over multiple lines. Break the line before the \`.

Reply:
Done.